### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by rathorevaibhav**
#520 

Changes include:

1. New welcome page at `/` route.

### CODE REVIEW

File: app/Http/Controllers/WelcomeController.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class WelcomeController extends Controller
+{
+    public function index()
+    {
+        if (auth()->check()) {
+            return redirect('home');
```
Comment (pankaj-ag) [inline app/Http/Controllers/WelcomeController.php]: Just Curious can't we use guest or auth middleware?

File: resources/views/welcome.blade.php [hunk 2]
```diff
@@ -1 +1,130 @@
...
+
+        <title>{{ config('app.name', 'Laravel') }}</title>
+        <link rel="icon" href="{{ asset('favicon.png') }}" sizes="32x32" />
+
+        <!-- Fonts -->
+        <link href="https://fonts.googleapis.com/css?family=Raleway:100,600" rel="stylesheet" type="text/css">
+        <link href="{{ mix('css/app.css') }}" rel="stylesheet">
+
+        <!-- Styles -->
+        <style>
```
Comment (pankaj-ag) [inline resources/views/welcome.blade.php]: Just a question why we are not using css/app.css or welcome.css for this? I think that will help us to use SAAS.

File: app/Http/Controllers/WelcomeController.php (hunk 1 above)
Comment (rathorevaibhav) [reply app/Http/Controllers/WelcomeController.php]: We need to redirect if the user is already logged-in the welcome page shouldn't appear (for now). He should be directed to the homepage. Not very sure how I can define the default route for logged-in users using a middleware.

**Comment by rathorevaibhav** [comment]
@pankaj-ag done with the requested changes. Ready for review again.

File: app/Http/Controllers/WelcomeController.php (hunk 1 above)
Comment (pankaj-ag) [reply app/Http/Controllers/WelcomeController.php]: I think you can. Found this in the documentation for guest middleware.
 
image

**Pull request merged**